components/common.js

```diff
@@ -0,0 +1,57 @@
+import {
+  Indent,
+  IndentationTypes,
+  withIndendation,
+} from "@asyncapi/generator-react-sdk";
+
+export function HTML({ childrenContent }) {
+  return `
+<!DOCTYPE html>
+<html lang="en">
+${childrenContent}
+</html>
+`;
+}
+
+export function Head({ title, cssLinks = [], styleContent = [],base }) {
+  const links = cssLinks
+    .map((link) => `<link rel="stylesheet" href="${link}">\n`)
+    .join("");
```

Comment:
If you make `join` then you transform `links` array to string, so then you cannot use `links` as array.